services/cad-service/nesting/engine.py

```
@@ -0,0 +1,166 @@
+"""Lightweight SVG nesting helpers compatible with OpenNest/SVGNest outputs."""
+
+from __future__ import annotations
+
+import xml.etree.ElementTree as ET
+from dataclasses import dataclass
+from typing import Iterable, List, Sequence
```

Ruff (0.14.1)

Import from `collections.abc` instead: `Iterable`, `Sequence`

Import from `collections.abc`

(UP035)